### ISSUE TITLE
ESC key should hide tooltip for accessibility

### ISSUE THREAD

**Issue by marchingband** [atomiks/tippyjs-react#386]
I achieved it like this, but it should be built in, it is an accessibility requirement.
```
  const [hide, setHide] = useState(false)
  const onKeyPress = useCallback(e=>{
    if(e.key == "Escape"){
      setHide(true)
    }
  }, [hide, setHide])
  const handleShow = useCallback(()=>{
    document.addEventListener("keydown", onKeyPress, true)
  }, [onKeyPress])
  const handleHide = useCallback(()=>{
    setHide(false)
    document.removeEventListener("keydown", onKeyPress, true)
  }, [onKeyPress, setHide])
```
